spec/Powertrain/TractionBattery.vspec

```diff
@@ -273,69 +331,28 @@ Charging.MaximumChargingCurrent.Phase3:
   unit: A
   description: Maximum AC charging current (rms) at inlet for Phase 3 that can be accepted by the system.
 
-Charging.ChargePortFlap:
-  deprecation: V4.1 - Replaced with Charging.IsChargePortFlapOpen
-  datatype: string
-  type: actuator
-  allowed: ['OPEN', 'CLOSED']
-  description: Status of the charge port cover(s), can potentially be controlled manually.
-               OPEN if at least one is open.
-
 Charging.IsChargePortFlapOpen:
+  deprecation: V4.3+ - Replaced with Charging.ChargingPort.IsFlapOpen to allow multiple charging ports.
```

Comment:
Shoudl we mark in the deprecation Note that it is instantiated now. Otherwise if somebody processes this automatically the results may be unexpected 

Suggestions

```
Charging.ChargingPort.*.IsFlapOpen
```

Or 

```
Charging.ChargingPort.<INSTANCE>.IsFlapOpen
```

Reply:
Thanks for the observation. I could add, no problem. However, that deprecation message indicates that one must see the `Charging.ChargingPort.IsFlapOpen` instead. In that part of the specification, the instantiation of the branch is visible. 

I agree with the fact that the instantiation is not clear in the current naming convention. I actually started recently a [discussion and proposed an option to improve the human-friendly name of VSS properties](https://github.com/COVESA/vehicle_signal_specification/issues/749). 

The problem I see is that the descriptions, comments, deprecations, are not following a strict style or format. Therefore, even if I make the suggested change here, the specification will remain to be inconsistent. I think a (semi)automated method to curate the specification would be ideal.

Reply:
I updated the deprecation note to make it the new instantiation explicit.